build/src/push.js

```diff
@@ -126,11 +126,14 @@ async function pushImage(definitionId, repo, release, updateLatest,
 
             if (replaceImage || !await isDefinitionVersionAlreadyPublished(definitionId, release, registry, registryPath, variant)) {
 
+                let skipPersistingCustomizationsFormFeatures = false;
```

Comment:
Typo in variable name: form -> from. Please fix here and below.
```suggestion
                let skipPersistingCustomizationsFromFeatures = false;
```